whitepapers/0014_transfer_verifier.md

```diff
@@ -160,3 +169,5 @@ Cancel is enabled, the suspicious transfers may become released early.
 
 It should be noted that this only affects small transfers, as Flow Cancel does not affect transfers over the big
 transfer limit.
+
+More details about the Flow Cancel feature can be found in the [Governor whitepaper](./0007_governor.md).
```

Comment:
I would either link above under "(See Governor whitepaper for more details)" or remove that string in the earlier paragraph now that you've added this last sentence